src/DonationForms/DataTransferObjects/DonateFormRouteData.php

```diff
@@ -134,4 +136,20 @@ public function toArray(): array
     {
         return get_object_vars($this);
     }
+
+    /**
+     * @unreleased
+     */
+    private function isValidForm(DonationForm $form): bool
+    {
+        if ($form->status->isTrash()) {
+            return false;
+        }
+
+        if (!$form->status->isPublished() && !current_user_can('edit_give_forms')) {
```

Comment:
I just want to confirm that your intention here is to allow admin users to still be able to donate on unpublished forms.

Reply:
@pauloiankoski yes that's correct or at least the intention. Published forms should allow public requests.  If the form is in pending/draft/private mode than it should only be used by an admin who is likely using for testing.